openid-sharedsignals-framework-1_0.md

```diff
@@ -2013,6 +2017,9 @@ The CAEP use cases that set the requirements are described in CAEP Use Cases (TO
 This section provides SSF profiling specifications for the Security Event Token (SET)
 {{RFC8417}} spec.
 
+{{RFC8417}} includes the usage of the `txn` claim in the specification. Transmitters SHOULD set the `txn` value in the Security Event Tokens (SETs). If the value is present, it MUST be unique to the underlying event that caused the Trasmitter to generate the Security Event Token (SET). The Transmitter, however, may use the same `txn` value across different Security Events Tokens(SETs), such as session revoked and credential change, to indicate that they originated from the same underlying cause or reason. 
```

Comment:
The text looks good, but I wonder if it should be moved down to near section 10.2.2 or so, with the specifications about the `iss` claim, the `exp` claim, the `aud` claim, and the `events` claim?

Reply:
Open to it. I placed it where I did because this was the closest to the SET Profile. Atul, did say he liked the placement of it, so I am not married to the placement.

Reply:
Done :)